api/pom.xml

```diff
@@ -239,13 +239,13 @@
                 <version>2.8</version>
                 <configuration>
 		    <additionalparam>-Xdoclint:none</additionalparam>
-                    <doctitle>Concurrency Utilities for Java EE ${project.version} API Specification</doctitle>
+                    <doctitle>Jakarta Concurrency ${project.version} Specification</doctitle>
                     <bottom>
-<![CDATA[Copyright &#169; 1996-2013,
+<![CDATA[Copyright &#169; 2019,
     <a href="http://www.oracle.com">Oracle</a>
-    and/or its affiliates. All Rights Reserved.
+    and/or its affiliates.
     Use is subject to
-    <a href="{@docRoot}/doc-files/speclicense.html" target="_top">license terms</a>.
+    <a href="{@docRoot}/doc-files/speclicense.html target="_top">license terms</a>.
```

Comment:
Was the " character  (end quote for href) accidentally removed on this line?

Reply:
yep good catch

Reply:
fixed